src/App.tsx

```diff
@@ -1,42 +1,36 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tab } from './types/type';
+import { Tabs } from './components/Tabs';
 
-export const tabs = [
+export const tabs: Tab[] = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
+
+  const getSelectedTab = tabs.find(tab => tab.id === selectedTabId) || null;
```

Comment:
this one stores not the function, but the selected tab
```suggestion
  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || null;
```